functions/packages/getVPCInfo/info_lambda.js

```diff
@@ -0,0 +1,82 @@
+var aws = require('aws-sdk');
```

Comment:
We you able to submit this as lambda.zip (A zip file including this single info_lambda.js file)?

Reply:
ok. I can help

Reply:
thanks!